### PR TITLE
Making a signup page

### PR DESCRIPTION
**Pull request by reidstivers**
added first and last name to the sign up realized we had no names in the user model and migrated a new one

also added strong params for names

also changed the head to a capital S in sitters

### CODE REVIEW

File: app/views/devise/confirmations/new.html.erb
Comment (rachel-schlaifer) [inline app/views/devise/confirmations/new.html.erb:1]: I was under the impression that we're using fake emails for this app - do we want the emails to be :confirmable?

**Review by rachel-schlaifer — CHANGES_REQUESTED** [review]

File: app/views/devise/confirmations/new.html.erb
Comment (reidstivers) [reply app/views/devise/confirmations/new.html.erb:1]: No, they don't need to be confirmable. Still commented out in this merge

Comment (reidstivers) [reply app/views/devise/confirmations/new.html.erb:1]: devise creates a lot of views on its own, we just need to use the ones we need

File: app/views/pages/home.html.erb
[the remark this answers is not in the document]
Comment (reidstivers) [reply app/views/pages/home.html.erb:1]: correct, this is the default page made by rails we were just using as a placeholder we will change to the listing page once we have it and a working navbar

File: app/views/devise/confirmations/new.html.erb
Comment (rachel-schlaifer) [reply app/views/devise/confirmations/new.html.erb:1]: Gotcha.

File: app/views/pages/home.html.erb
[the remark this answers is not in the document]
Comment (rachel-schlaifer) [reply app/views/pages/home.html.erb:1]: Sounds good.

**Review by rachel-schlaifer — APPROVED** [review]

File: app/views/pages/home.html.erb
[the remark this answers is not in the document]
Comment (reidstivers) [reply app/views/pages/home.html.erb:1]: I just made a sign up/log out button for convenience and testing puposes

**Pull request merged**